mix.exs

```diff
@@ -60,7 +60,7 @@ defmodule ElixirBoilerplate.Mixfile do
       {:absinthe_error_payload, "~> 1.0"},
 
       # Database
-      {:ecto_sql, "~> 3.2"},
+      {:ecto_sql, "~> 3.3"},
```

Comment:
Is this change part of the New Relic integration?

Reply:
yes, New Relic Agent expected `ecto_sql ~> 3.3`https://hex.pm/packages/new_relic_agent/1.17.0